src/plugins/intel_gpu/src/graph/graph_optimizer/handle_input_padding.cpp

```diff
@@ -1,129 +0,0 @@
-// Copyright (C) 2018-2023 Intel Corporation
```

Comment:
Maybe I missed something, but is async padding is supported by many kernels? (AFAIK it isn't..is it?)
If only ref kernel support async padding, is it better to use ref kernel than opt kernel with borders?

Reply:
I think pad_end is not really needed in the kernels if we know output shape. Anyway, kernel selector parameters for convolution don't even have second parameter for pad, it means that none of the kernels use pad_end explicitly (even ref)

Reply:
I have thought that the reason why the handle_input_padding exists is because the kernels are not supporting asymmetric padding. But you mean that it is not true, right? If so the original pass was redundant.. I think it would be good to check the benchmark test.